tools/launchdev.sh

```diff
@@ -68,6 +68,7 @@ function init(){
     fi
 
     VIRTUALENV=${VIRTUALENV_DIR:-${ST2_REPO}/virtualenv}
+    VIRTUALENV=$(readlink -f ${VIRTUALENV})
```

Comment:
I assume this is to accommodate symlink in your dev env.  Will this break existing setup for us?

Reply:
It's so an absolute path is used everywhere (just resolves relative path to an absolute one). Everything should still work the same.